MapboxNavigation/PollyVoiceController.swift

```diff
@@ -68,14 +69,13 @@ public class PollyVoiceController: RouteVoiceController {
         guard shouldSpeak(for: notification) == true else { return }
```

Comment:
Move everything below here into `speak(_:)` and delete this method implementation, including this `guard` statement. It should no longer be necessary for PollyVoiceController to override this method.